aten/src/ATen/native/cuda/Blas.cpp

```diff
@@ -672,4 +680,64 @@ TORCH_IMPL_FUNC(addmv_out_cuda)(const Tensor &self, const Tensor &mat, const Ten
   }
 }
 
+
+void _int_mm_out_cuda(const Tensor& self, const Tensor& mat2, Tensor& result) {
+  TORCH_CHECK(self.size(0) > 16, "self.size(0) needs to be greater than 16, but got ", self.size(0));
+  TORCH_CHECK(self.size(1) > 0 && self.size(1) % 8 == 0, "self.size(1) needs to be greater than 0 and a multiple of 8, but got ", self.size(1));
```

Comment:
does cublas require multiple of 8?

Reply:
For int8 it seems to. I tried a wide scan across all shapes [1, 256] for m, n, k.